tests/linearizability/linearizability_test.go

```diff
@@ -67,6 +76,27 @@ func TestLinearizability(t *testing.T) {
 				ClusterSize:   1,
 				GoFailEnabled: true,
 			},
+			traffic: trafficConfig{
+				minimalQPS:  100,
+				maximalQPS:  200,
+				clientCount: 8,
+				traffic:     PutGetTraffic,
+			},
+		},
+		{
+			name:      "Issue13766",
+			failpoint: KillFailpoint,
+			config: e2e.EtcdProcessClusterConfig{
+				ClusterSize:         3,
+				InitialCorruptCheck: true,
+			},
+			skipValidation: true,
```

Comment:
If you skip the validation, then it isn't linearizablity test anymore. So it doesn't make sense to get the test included in the linearzability test suite. 

And also shouldn't we check HashKV after the test? The existing functional test already covers the case. Linearizablity test is good, but please do not recreate the wheel. Instead, I would suggest to enhance both the linearizablity and functional tests.

Reply:
> Linearizablity test is good, but please do not recreate the wheel. Instead, I would suggest to enhance both the linearizablity and functional tests.

As long functional tests are flaky they are useless. Using  initial hash check in linerazibility tests I see as a shortcut that I hope to remove in future, however I don't see same possibility in functional tests. 

We are already not benefiting from running functional tests, so when we reproduce all standing inconstancy issues and add network blackholing in lineariziability tests I will propose to remove them.